cli/src/benchmark/mod.rs

```diff
@@ -208,6 +208,16 @@ impl BenchmarkCommands {
 						&client,
 					);
 
+					let getter = Getter::trusted(
+						TrustedGetter::free_balance(new_account.public().into())
+							.sign(&KeyPair::Sr25519(new_account.clone())),
+					);
+					let getter_start_timer = Instant::now();
+					if get_state(&client.client_api, shard, &getter) == None {
+						warn!("Getter returned None");
+					}
+					info!("Getter execution took {} ms", getter_start_timer.elapsed().as_millis());
```

Comment:
Added a balance getter to the benchmark execution, in addition to the balance transfer call. The getter execution timing is then logged (adjust log level for this module to print them).